dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/navigation/NavTool.java

```diff
@@ -69,6 +69,23 @@ protected NavResult getNav(Host host, String path) throws DotDataException, DotS
         return getNav(host, path, this.currentLanguage, this.systemUser);
     }
 
+    protected void setItemLinkValues(NavResult nav,Link itemLink, List<NavResult> children){
+        if (itemLink.getLinkType()
```

Comment:
I think the condition should be in the reverse order
first check null and them equals